### DISCUSSION TITLE
How to instantly remove a server that no longer exists

### DISCUSSION THREAD

**Discussion by MarlonHeiber** [gravitational/teleport#18384] [Q&A] [answered]
When a server that had the teleport agent installed is destroyed it continues to appear in the teleport server list. The server stays there for several minutes on the list until it is automatically removed by the teleport.

The problem is that people still trying to connect to the server that no longer exists because it is still being listed.

How can i instantly remove a server that no longer exists from the list?

**Comment by webvictim** [comment, marked as answer]
- Run `tsh ls -v` or `tctl get nodes` to get the server's UUID.
- Run `tctl rm node/<uuid>` to delete it from the backend.
